tsdb/challenges/default.json

```diff
@@ -120,4 +120,123 @@
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          }
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "tsdb",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('green')}}",
+              "wait_for_no_relocating_shards": "true"
+            },
+            "retry-until-success": true
+          }
+        },
+        {
+          "operation": "index",
+          "warmup-time-period": 240,
+          "clients": {{bulk_indexing_clients | default(8)}},
+          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+        },
+        {
+          "name": "refresh-after-index",
+          "operation": "refresh"
+        },
+        {
+          "operation": {
+            "operation-type": "force-merge",
+            "request-timeout": 7200{%- if force_merge_max_num_segments is defined %},
+            "max-num-segments": {{ force_merge_max_num_segments | tojson }}
+            {%- endif %}
+          }
+        },
+        {
+          "name": "wait-until-merges-finish",
+          "operation": {
+            "operation-type": "index-stats",
+            "index": "_all",
+            "condition": {
+              "path": "_all.total.merges.current",
+              "expected-value": 0
+            },
+            "retry-until-success": true,
+            "include-in-reporting": false
+          }
+        },
+        {
+          "name": "refresh-after-force-merge",
+          "operation": "refresh"
+        },
+        {
+          "name": "block-source-index-writes",
+          "operation": {
+            "operation-type": "raw-request",
+            "path": "/tsdb/_block/write",
+            "method": "PUT"
+          }
+        },
+        {
+          "name": "downsampling-1m",
+          "operation": {
+            "operation-type": "downsampling",
+            "body": {
+              "fixed-interval": "1m",
+              "source-index": "tsdb",
+              "target-index": "tsdb-1m"
+            }
+          }
+        },
+        {
+          "name": "downsampling-1h",
+          "operation": {
+            "operation-type": "downsampling",
+            "body": {
+              "fixed-interval": "1h",
+              "source-index": "tsdb",
+              "target-index": "tsdb-1h"
+            }
+          }
+        },
+        {
+          "name": "downsampling-1d",
+          "operation": {
+            "operation-type": "downsampling",
+            "body": {
+              "fixed-interval": "1d",
+              "source-index": "tsdb",
+              "target-index": "tsdb-1d"
+            }
+          }
+        },
+        {
+          "operation": "date-histo-entire-range-1m",
```

Comment:
I added a new one because the index they tun on is the result of downsampling, while in this challenge we need to run it on the original index. Is that what you need?

Reply:
Ah! I see - this is for the entire range either way. Ok. Ignore this comment.